src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "[a-z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,4}$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email.isEmpty()) {

```

Comment:
what is it for? `pattern.matcher("");` will return true?

![image](https://user-images.githubusercontent.com/20326858/225882358-157891c2-7428-4999-9bd8-bdc65c45747e.png)